sw/device/silicon_creator/rom/BUILD

```diff
@@ -175,7 +175,6 @@ opentitan_binary(
     linker_script = ":linker_script",
     deps = [
         ":rom_without_keys",
-        "//sw/device/silicon_creator/rom/keys/fake/rsa",
```

Comment:
do we need the `rom_with_fake_keys` target now that keys are in OTP? maybe we can remove it in a follow up and just rename `rom_without_keys` to `rom`?